docs/source/capabilities/algorand-client.md

```diff
@@ -39,7 +34,7 @@ algorand = AlgorandClient.from_config(
 Once you have an `AlgorandClient` instance, you can access the SDK clients for the various Algorand APIs via the `algorand.client` property.
 
 ```python
-algorand = AlgorandClient.default_localnet()
+algorand = AlgorandClient.default_local_net()
```

Comment:
AlgorandClient.default_localnet()

Reply:
@neilcampbell isn't it a bit inconsistent, given that we also have `mainnet` `testnet` methods on the client? Why would localnet specifically have `local_net`.